### ISSUE TITLE
Water level exchanges between ocn&wav in coastal_ike_shinnecock_atm2sch2ww3

### ISSUE THREAD

**Issue by yunfangsun** [oceanmodeling/ufs-weather-model#121]
Hi @uturuncoglu @pvelissariou1 @josephzhang8 @saeed-moghimi-noaa @janahaddad ,

In the RT case of coastal_ike_shinnecock_atm2sch2ww3, the following variables are used:

```
(merge_to_ocn): Sa_u10m(ocn) = Sa_u10m(atm)
(merge_to_ocn): Sa_v10m(ocn) = Sa_v10m(atm)
(merge_to_ocn): Sa_pslv(ocn) = Sa_pslv(atm)
(merge_to_ocn): Sw_wavsuu(ocn) = Sw_wavsuu(wav)
(merge_to_ocn): Sw_wavsuv(ocn) = Sw_wavsuv(wav)
(merge_to_ocn): Sw_wavsvv(ocn) = Sw_wavsvv(wav)
(merge_to_wav): Sa_u10m(wav) = Sa_u10m(atm)
(merge_to_wav): Sa_v10m(wav) = Sa_v10m(atm)
(merge_to_wav): So_u(wav) = So_u(ocn)
(merge_to_wav): So_v(wav) = So_v(ocn)
```
and from @saeed-moghimi-noaa 

`I think we followed wave coupling for large-scale deep ocean settings. for 2D coupling, we need to include WL. Please make a note or start an issue to address this. Then I argue the wave coupling is not complete! In particular for landfalling hurricanes`

@uturuncoglu do we support the water level exchange in our current version of ufs-coastal?

Thank you!

**Comment by uturuncoglu** [comment]
@yunfangsun No, we are not exchanging water level with any component. That is totally new for me. @janahaddad If you don't mind could you add an item to our next meeting to discuss about it.

**Comment by josephzhang8** [comment]
Related: wet/dry masks in ocean should be passed onto WW3? Thx

**Comment by uturuncoglu** [comment]
@josephzhang8 exchanging wet and dry mask can be added but I am not sure how CMEPS will handle it. At this point, CMEPS gets ocean mask in the data initialization and keep constant. Anyway, that needs to be explored.

**Comment by saeed-moghimi-noaa** [comment]
The most needed is the WL exchange. Without that we can not go ahead with 2D coupling. I suggest we open an issue for the mask and get back to it later.

**Comment by uturuncoglu** [comment]
@saeed-moghimi-noaa Okay. I think the title of this issue is wrong. At this point, `coastal_ike_shinnecock_atm2sch2ww3` is three way could system that WW3 and SCHSIM is coupled in 2d way. So, I am not sure why `coastal_ike_shinnecock_atm2sch2ww3` required WL at this point. So, if WL one is different configuration and it requires implementing another RT. Right? So, maybe the title of this issue might be changed. 

As we discussed before, we need a test case for 3d coupled system to develop the mission wiring. I think @josephzhang8 and others are working on it. After we have the case, we could start to implement it. maybe WL could be the first filed that needs to be sent from ocean to wave (I hope I am right on it). And then, this implementation can be used by others as a reference to add other variables to finalize the work related to the 3d coupling. 

For wet-dry mask, maybe we could assume that everywhere in the ocean domain is wet in terms of CMEPS involvement. Then, once we sent the files along with wet-dry mask to components (i.e. wave), component just get the data over ocean by filtering the incoming data with the mask. Anyway, we could discuss it more in our next meeting.

**Comment by josephzhang8** [comment]
@uturuncoglu: regarding 3D coupling, I'm still waiting for you to show me a working example of adding new variables in WW3 and export them. Thx.

**Comment by uturuncoglu** [comment]
@josephzhang8 As I remember from our last meeting (please correct me if I am wrong), there was a 3d test configuration in schism side but working with another wave model and the plan was to modify that case to work with ww3. Without having a working configuration, it would be hard to test the modifications and create RT for it. If you want we could have hands-on session after UFS meeting and I could show which part of the code in WW3 cap needs to be changed (incl. required CMEPS changes to send those new variables from wave to ocean).

**Comment by josephzhang8** [comment]
Thx @uturuncoglu. I sent the model setup to Ali for him to set up WW3, but have not heard back. I'll send him a reminder. Thx

**Comment by yunfangsun** [comment]
https://github.com/NOAA-EMC/WW3/discussions/1280

**Comment by janahaddad** [comment]
Thanks @yunfangsun .... for full update: we discussed the WL exchange issue at today's tag-up. @uturuncoglu pointed us to [this commented line in wav_import_export.f90](https://github.com/oceanmodeling/WW3/blob/e8b3ef615eb332198846cd7848d093672504cf30/model/src/wav_import_export.F90#L109). He's initiated discussion with EMC WW3 devs about why this was commented out, and he'll try uncommenting to activate/test

**Comment by janahaddad** [comment]
This issue is superseded by others in #95
